db_upgrade_scripts/mosip_pms/upgrade.properties

```diff
@@ -1,4 +1,8 @@
 MOSIP_DB_NAME=
+SOURCE_DB1_NAME=mosip_authdevice
+SOURCE_DB2_NAME=mosip_regdevice
+SOURCE_DB1_SUPPORT_FILE=sql/1.2.0_pms-authdevice-support.sql
```

Comment:
file names needs to be updated

Reply:
renamed the value of property as suggested